services/app/pawnguild/registration/templates/confirmation_email.txt

```diff
@@ -1,3 +1,16 @@
-Thanks for registering at PawnGuild!
-Copy paste the link into your browser to confirm your account. Once confirmed, you can create Pawns
-{{ link }}
\ No newline at end of file
+-------------------------------------------------
+Activate your PawnGuild Account
+-------------------------------------------------
+
+Thank you for registering on PawnGuild, {{ user.username }}!
+
+In order to create and manage pawns, please copy and paste the following link in to your browser to activate your account.
+
+{{ link }}
+
+This link expires on {{ expiry|time:"TIME_FORMAT" }}
```

Comment:
The time is using UTC time. It looked weird to me when it should have said 1:36 pm for me, but said 1:36 am. Let's either indicate it's UTC by just writing out UTC in the email right after the time or just say "This link expires in 1 hour" to avoid the timezone issue

Reply:
Changing to `This link expires in 1 hour.`